pkg/virt-handler/devices/mandatory.go

```diff
@@ -0,0 +1,103 @@
+package devices
+
+import (
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"kubevirt.io/kubevirt/pkg/api/v1"
+	"kubevirt.io/kubevirt/pkg/virt-handler/isolation"
+)
+
+const (
+	kvm = "/dev/kvm"
+	tun = "/dev/net/tun"
+)
+
+type KVM struct {
+}
+
+type TUN struct {
+}
+
+// Setup creates /dev/kvm inside a container and sets the right permissions for qemu
+func (k *KVM) Setup(_ *v1.VirtualMachineInstance, hostNamespaces *isolation.IsolationResult, podNamespaces *isolation.IsolationResult) error {
```

Comment:
These two plugins only need access to the mount namespace of the pod, so no need to do something with the network namespace.

Entering the container network namespace would work like this:

```
n, err := ns.GetNS(podNamespaces.NetNamespace())
err = n.Do(func(...) error{
    do something in container
})
```